tsstats/utils.py

```diff
@@ -49,3 +49,22 @@ def filter_threshold(clients, threshold):
     :rtype: list
     '''
     return list(filter(lambda c: c[1] > threshold, clients))
+
+
+def flip_ident_map_list(identmap):
```

Comment:
I'm currently not really sure about the function-name, but I guess we can talk about that later.